### PR TITLE
Fix ControlNetModel.from_unet do not load add_embedding

### PR DESCRIPTION
**Pull request by laksjdjf**
# What does this PR do?
Recreate from #7173 

## Who can review?

@sayakpaul , @yiyixuxu

### CODE REVIEW

File: src/diffusers/models/controlnet.py [hunk 1]
```diff
@@ -508,6 +508,9 @@ def from_unet(
 
             if controlnet.class_embedding:
                 controlnet.class_embedding.load_state_dict(unet.class_embedding.state_dict())
+
```
Comment (laksjdjf) [inline src/diffusers/models/controlnet.py:511]: Is this line unnecessary?

**Comment by sayakpaul** [comment]
Let's fix the tests :)

**Comment by laksjdjf** [comment]
> Let's fix the tests :)

@sayakpaul
I removed meaningless spaces for test.
Are there any other operations required?

**Review by sayakpaul — APPROVED** [review]
Looks good to me!

**Comment by laksjdjf** [comment]
@sayakpaul 
Is this error relevant to this pr?
```
=========================== short test summary info ============================
FAILED tests/pipelines/unidiffuser/test_unidiffuser.py::UniDiffuserPipelineFastTests::test_attention_slicing_forward_pass - requests.exceptions.ReadTimeout: (ReadTimeoutError("HTTPSConnectionPool(host='huggingface.co', port=443): Read timed out. (read timeout=10)"), '(Request ID: 869e35f5-5627-4acf-9853-7187ce7d0656)')
==== 1 failed, 2193 passed, 989 skipped, 4415 warnings in 937.44s (0:15:37) ====
```

**Comment by sayakpaul** [comment]
It's not!

**Pull request merged**